recipe_batches/recipe_batches.py

```diff
@@ -3,12 +3,29 @@
 import math
 
 def recipe_batches(recipe, ingredients):
-  pass 
+  whole_batches = {}
+  for rkey in recipe.items():
```

Comment:
double nested loop here! it's working, so leave this for after you're done with all of MVP, but see if you can make this O(n) instead of O(n**2)